trezorlib/tests/device_tests/test_msg_cardano_get_address.py

```diff
@@ -25,14 +25,14 @@
 @pytest.mark.cardano
 @pytest.mark.skip_t1  # T1 support is not planned
 class TestMsgCardanoGetAddress(TrezorTest):
-
-    def test_cardano_get_address_v1(self):
+    @pytest.mark.parametrize("path,expected_address", [
+        ("m/44'/1815'/0'/0/0", "2w1sdSJu3GVfqnGAyqAdaWrN8Txv1vCZTN1Pe2AA54ysjWNbNzma3WVtSJfMc6HpM9KEQsdJ7oALPwfQWesRp8QDsFRQpzuNrdq"),
+        ("m/44'/1815'/0'/0/1", "2w1sdSJu3GVhMmEYeGYEPWahV1V17pFw59GfgqjSRqa6x1rKFxbyCZrQWLe78xdSx3zyed6DrrN5yMgoY7ST2vJeaMzUDB7W3WG"),
+        ("m/44'/1815'/0'/0/2", "2w1sdSJu3GVeHCDfy3mjq8RkzkN3Vh7Di3cB8NRzkwkLQ2FAjxX1kvkNdP9hNBzyBVEJdeWwyb5GfFYXgKe7rPgvWj2QD8FE4W3"),
+    ])
+    def test_cardano_get_address_v1(self, path, expected_address):
         # data from https://iancoleman.io/bip39/#english
         self.setup_mnemonic_nopin_nopassphrase()
 
-        address = get_address(self.client, parse_path("m/44'/1815'/0'/0/0")).address
-        assert address == '2w1sdSJu3GVfqnGAyqAdaWrN8Txv1vCZTN1Pe2AA54ysjWNbNzma3WVtSJfMc6HpM9KEQsdJ7oALPwfQWesRp8QDsFRQpzuNrdq'
-        address = get_address(self.client, parse_path("m/44'/1815'/0'/0/1")).address
-        assert address == '2w1sdSJu3GVhMmEYeGYEPWahV1V17pFw59GfgqjSRqa6x1rKFxbyCZrQWLe78xdSx3zyed6DrrN5yMgoY7ST2vJeaMzUDB7W3WG'
-        address = get_address(self.client, parse_path("m/44'/1815'/0'/0/2")).address
-        assert address == '2w1sdSJu3GVeHCDfy3mjq8RkzkN3Vh7Di3cB8NRzkwkLQ2FAjxX1kvkNdP9hNBzyBVEJdeWwyb5GfFYXgKe7rPgvWj2QD8FE4W3'
+        address = get_address(self.client, parse_path(path)).address
```

Comment:
ISTM this will fail now that `get_address` returns the address directly?

Reply:
Yes it failed. Fixed. Also replaced custom mnemonic for all all...